manifests/0000_08_cluster-dns-operator_02-deployment.yaml

```diff
@@ -31,6 +31,8 @@ spec:
               value: openshift/origin-coredns:v4.0
             - name: OPENSHIFT_CLI_IMAGE
               value: openshift/origin-cli:v4.0
+            - name: OPERATOR_IMAGE_VERSION
+              value: openshift/origin-cluster-dns-operator:latest
```

Comment:
Is this not same as  `Spec.Template.Spec.Containers[0].Image`?
https://github.com/openshift/cluster-dns-operator/blob/a15bd50584801db025616e74affa91ce88f37937/manifests/0000_08_cluster-dns-operator_02-deployment.yaml#L25

Reply:
If there are the same always then we don't need another env. Creates confusion if both differs when we expect them to be the same value.

Reply:
Actually the value should have been `0.0.0_version_cluster-dns-operator`.  Fixed—thanks!